sensing/pointcloud_preprocessor/schema/crop_box_filter.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameter for crop box filter",
+  "type": "object",
+  "definitions": {
+    "crop_box_filter": {
+      "type": "object",
+      "properties": {
+        "min_x": {
+          "type": "number",
+          "description": "x-coordinate minimum value for crop range",
+          "default": "-1.0"
+        },
+        "min_y": {
+          "type": "number",
+          "description": "y-coordinate minimum value for crop range",
+          "default": "-1.0"
+        },
+        "min_z": {
+          "type": "number",
+          "description": "z-coordinate minimum value for crop range",
+          "default": "-1.0"
+        },
+        "max_x": {
+          "type": "number",
+          "description": "x-coordinate maximum value for crop range",
+          "default": "1.0"
+        },
+        "max_y": {
+          "type": "number",
+          "description": "y-coordinate maximum value for crop range",
+          "default": "1.0"
+        },
+        "max_z": {
+          "type": "number",
+          "description": "z-coordinate maximum value for crop range",
+          "default": "1.0"
+        }
```

Comment:
add negative parameter

Reply:
Sorry, I didn't get it. Is that a disciption or, or the value for paramter?

Reply:
The parameter that named "negative" which is in the crop box node.

Reply:
@Ariiees  
After you fix this one, please don't click 'Resolve conversation,' thanks. 
Also, it would be appreciated if you could post the commit here for me to review easily.

Reply:
Oh, I see. Added!